src/FunctionLoader.ts

```diff
@@ -20,13 +20,13 @@ export class FunctionLoader implements IFunctionLoader {
         };
     } = {};
 
-    async load(functionId: string, metadata: rpc.IRpcFunctionMetadata): Promise<void> {
+    async load(functionId: string, metadata: rpc.IRpcFunctionMetadata, packageJson: Object = {}): Promise<void> {

```

Comment:
I assume the only reason you made it optional was to make the test code easier, but test code should not affect src code like this. It would be a bug if we forgot to pass `packageJson` to `load` in src code, so the parameter should be required

Reply:
Busted haha, addressed this now